src/lib/events/enums.json

```diff
@@ -110,6 +110,10 @@
                         "6": {
                             "name": "no_rc_and_no_datalink",
                             "description": "No RC and no datalink"
+                        },
+                        "7": {
+                            "name": "no_gps",
+                            "description": "No valid GPS"
```

Comment:
We prob need to make the go through the whole code base and translate GPS to GNSS at some point, but it's prob best done in one piece than piecewise, so that's not a blocker for this Pr.

Reply:
Good idea, continuing to call it GPS everywhere is a bit ignorant.